tests/builtins/test_abs.py

```diff
@@ -8,7 +8,8 @@ class NotAbsLike:
                 pass
             x = NotAbsLike()
             print(abs(x))
-            """)
+            print('Done.')
+            """, exits_early=True)
```

Comment:
More generally, the "print('done')" statement that is common to many tests can now be removed, as we have a more reliable canary.